addons/mail/static/tests/message/message.test.js

```diff
@@ -361,6 +362,7 @@ test("Edit and click save", async () => {
     await insertText(".o-mail-Message .o-mail-Composer-input", "Goodbye World", { replace: true });
     await click(".o-mail-Message a", { text: "save" });
     await contains(".o-mail-Message-body", { text: "Goodbye World (edited)" });
+    await contains("[title='Today at 1:00 PM']");
```

Comment:
Would target `em:contains('(edited)')[title='Today at 1:00 PM']` so there's less confusion this matches the title of create datetime.